server/broadcast.go

```diff
@@ -330,11 +337,6 @@ func transcodeSegment(cxn *rtmpConnection, seg *stream.HLSSegment, name string)
 			if res == nil && err == nil {
 				return errors.New("Empty response")
 			}
-			if shouldStopStream(err) {
```

Comment:
Any particular reason why this code block had to move up to `processSegment()`?

Reply:
If I'm not mistaken this code block didn't really surve a purpose before as `shouldStopStream()` would just always return false and there is nothing in the code block following the ìf` statement anyway. I just decided to reuse it for the purpose of checking the error and whether it is non-retryable.